R/get.R

```diff
@@ -376,152 +376,6 @@ content_list_guid_has_access <- function(content_list, guid) {
-#'     successful bundle deployment. Applies only to content with an app_mode
-#'     of rmd-static.
-#'   * `created_time`: The timestamp (RFC3339) indicating when this
-#'     content was created.
-#'   * `last_deployed_time`: The timestamp (RFC3339) indicating when
-#'     this content last had a successful bundle deployment performed.
-#'   * `r_version`: The version of the R interpreter associated
-#'     with this content. The value null represents that an R interpreter is
-#'     not used by this content or that the R package environment has not been
-#'     successfully restored. Automatically assigned upon the successful
-#'     deployment of a bundle.
-#'   * `py_version`: The version of the Python interpreter
-#'     associated with this content. The value null represents that a Python
-#'     interpreter is not used by this content or that the Python package
-#'     environment has not been successfully restored. Automatically assigned
-#'     upon the successful deployment of a bundle.
-#'   * `build_status`:
-#'   * `run_as`: The UNIX user that executes this content.
-#'     When null, the default Applications.RunAs is used. Applies
-#'     only to executable content types - not static.
-#'   * `run_as_current_user`: Indicates if this content is allowed
-#'     to execute as the logged-in user when using PAM authentication.
-#'     Applies only to executable content types - not static.
-#'   * `description`: A rich description of this content
-#'   * `app_role`: The relationship of the accessing user to this
-#'     content. A value of owner is returned for the content owner. editor
-#'     indicates a collaborator. The viewer value is given to users who are
-#'     permitted to view the content. A none role is returned for
-#'     administrators who cannot view the content but are permitted to view
-#'     its configuration. Computed at the time of the request.
-#'   * `owner_first_name`: The first name of the owner of the
-#'     content.
-#'   * `owner_last_name`: The last name of the owner of the
-#'     content.
-#'   * `owner_username`: The username of the owner of the
-#'     content.
-#'   * `owner_guid`: The unique identifier for the owner
-#'   * `owner_email`: The email of the content owner
-#'   * `owner_locked`: Is the owners user account locked?
-#'   * `is_scheduled`: Is this content scheduled?
-#'   * `git`: Is this content deployed via git or GitHub?
-#'
-#' @details
-#' Please see https://docs.posit.co/connect/api/#getContent for more
-#' information.
-#'
-#' @examples
-#' \dontrun{
-#' library(connectapi)
-#' client <- connect()
-#'
-#' get_content_old(client, limit = 20)
-#' }
-#'
-#' @export
-get_content_old <- function(src, filter = NULL, limit = 25, page_size = 25) {
-  validate_R6_class(src, "Connect")
-
-  lifecycle::deprecate_warn("0.1.0.9023", "get_content_old()", "get_content()", details = "The filter argument is deprecated and the structure of the response has changed with the public API")
```

Comment:
This is likely more a comment about the deprecation warning, but this makes it sound like _only_ the `filter` argument is deprecated? 

But given the function name, that must be wrong / I'm fine removing this.

I don't think this is worth keeping it for, but one note that #219 mentions this being a place one can acquire vanities without looping through

Reply:
I see the concern but I think this still should be removed.

* The deprecation warning says that `get_content_old()` is deprecated and `get_content()` is the new function you should call.
* The API doc link it has is broken. The API it's hitting is the "old" undocumented one.
* It is not tested.

We should solve the vanity URL issue in the API better.